source/_posts/front-commerce-2.2.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Release: Front-Commerce 2.2"
+date: 2020-11-18
+---
+
+Front-Commerce 2.2 has a heavy focus on Magento 1 features thanks to new clients migrating their Legacy Magento 1 shops on Front-Commerce. Read more about this version.
+
+<!-- more -->
+
+## Guest Checkout
+
+It is now possible to complete a Checkout in Front-Commerce without being logged. This is a frequently asked feature as it can really boost your sales. This also means that we've made all the changes necessary to the theme and that it will now be possible to add the Guest Checkout to Magento 2 with very few changes. Stay tuned!
```

Comment:
```suggestion
It is now possible to complete a Checkout in Front-Commerce without being logged. This is a frequently asked feature as it can really boost your sales. This also means that we've made all the changes necessary to the theme and that the Guest Checkout for Magento 2 will be supported in our next release. Stay tuned!
```

… sans prendre trop de risques niveau roadmap :joy: